src/components/GameView/GameCard.vue

```diff
@@ -79,6 +85,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    isThrees: {
```

Comment:
how about `isThreeTarget`
```suggestion
    isThreeTarget: {
```